iceoryx_examples/automotive_soa/src/minimal_proxy.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "minimal_proxy.hpp"
+
+constexpr char MinimalProxy::m_serviceIdentifier[];
+
+MinimalProxy::MinimalProxy(owl::kom::ProxyHandleType& handle) noexcept
+    : m_instanceIdentifier(handle.GetInstanceId())
+{
+    if (handle.GetServiceId() != owl::kom::ServiceIdentifier{iox::cxx::TruncateToCapacity, m_serviceIdentifier})
+    {
+        std::cerr << "Handle does not match MinimalProxy class. Can't construct MinimalProxy, terminating!"
+                  << std::endl;
+        std::terminate();
+    }
+}
+
+owl::kom::FindServiceCallbackHandle
+MinimalProxy::EnableFindServiceCallback(owl::kom::FindServiceCallback<owl::kom::ProxyHandleType> handler,
```

Comment:
Please add const to all arguments which are not modified. Here and in the remaining methods.